lib/addons/adot/index.ts

```diff
@@ -22,7 +22,7 @@ const defaultProps = {
  /**
   * Implementation of Adot Collector EKS add-on.
   */
-export class AdotCollectorAddOn extends CoreAddOn {
+ export class AdotCollectorAddOn extends CoreAddOn {
```

Comment:
why extra leading space? applies elsewhere

Reply:
Fixing this in AMP Addon PR.